neptune/new/exceptions.py

```diff
@@ -392,15 +392,21 @@ def __init__(self, label: str):
 
 class InactiveModelException(InactiveContainerException):
     resume_info = """
-    TODO: NPT-11349"""
+    - Resume the {container_type} to continue logging to it.
+    - Don't invoke `stop()` on a {container_type} that you want to access. If you want to stop monitoring only,
+    you can resume a {container_type} in read-only mode:
+    https://docs.neptune.ai/you-should-know/connection-modes#read-only"""
```

Comment:
simple placeholder (same for Model and ModelVersion):
![image](https://user-images.githubusercontent.com/14312755/153873087-7de8b432-a0cf-4887-8050-3eb6c22cbd74.png)

Reply:
2. Content for resuming:

```
    - Resume the model to continue logging to it:
    https://docs.neptune.ai/api-reference/neptune#.init_model
    ```

```
    - Resume the model version to continue logging to it:
    https://docs.neptune.ai/api-reference/neptune#.init_model_version
    ```

Reply:
1. Why `model_version` not "model version"?
2. In the second part there is still {container_type}

Reply:
Ad 2. Fixed, sorry
Ad 1. That's the value of our enum -- we use it in places where space is not really a safe option (e.g. filesystem paths). We're using variables to limit copy&paste, as we already have a lot of text in `exceptions.py`. Could you perhaps live with this underscore? :)

Reply:
I can and I should, I will always ask "maybe?" to push toward user :)